src/detect-engine-helper.c

```diff
@@ -81,6 +81,20 @@ int DetectHelperBufferMpmRegister(const char *name, const char *desc, AppProto a
     return DetectBufferTypeGetByName(name);
 }
 
+int DetectHelperMultiBufferMpmRegister(const char *name, const char *desc, AppProto alproto,
```

Comment:
these generic detect additions should be in their own commit

Reply:
ok